week5/src/hooks/useGetLetterHook.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { LetterProps } from '../types';
+
+export const useGetLetterHook = () => {
+  const [letters, setLetter] = useState<LetterProps[]>([]);
```

Comment:
아 여기서쓴거면 
```suggestion
 const [letterList, setLetterList] = useState<LetterInfo[]>([]);
```
가 적합하게따아